internal/projects/create.go

```diff
@@ -1,5 +1,5 @@
 //
-// Copyright 2023 Stacklok, Inc.
+// Copyright 2024 Stacklok, Inc.
```

Comment:
Nit: Do we need to update this date in all our files? 😄

Reply:
No, you normally just update them when you create a new file. In this case, I moved the whole file and updated this.